BugDoom - web app/src/main/java/com/infoshareacademy/pl/exception/TrackNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package com.infoshareacademy.pl.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
```

Comment:
czy tutaj nie powinno być value = HttpStatus.BAD_REQUEST?

Reply:
Na  ten moment nie jest nam to potrzebne